### DISCUSSION TITLE
Opencv autocomplete not working on Visual Studio Code

### DISCUSSION THREAD

**Discussion by Nabil845** [jasmcaus/caer#75] [Q&A]
Hello, 

Thanks to your tutorial video, I have install opencv module via the comand 'pip caer'. 

It's working but ... the autocomplete is not working. Do you know how I can fix that ?

**Comment by jasmcaus** [comment]
Hello @Nabil845. Most editors (VSCode too) won't provide autocomplete for OpenCV because OpenCV is a C++ library with Python bindings. Editors can only provide autocomplete/Intellisense for code written in Python. Hope this helps. `Caer` should have autocomplete though.
